client/src/main/scala/drt/client/components/FlightsWithSplitsTable.scala

```diff
@@ -192,11 +193,11 @@ object FlightTableRow {
           (None, props.originMapper(flight.Origin)),
           (None, s"${flight.Gate}/${flight.Stand}"),
```

Comment:
Gate/Stand needs to be unwrapped from their Options (currently displaying as Some(Whatever) on the front end.